string goal1
string goal2
string partID
float32 age
float32 score
float32 timestamp
bool reload
